java/release_notes.txt

```diff
@@ -1,3 +1,22 @@
+Sep 07, 2016: libphonenumber-7.6.1
+Code changes:
+ - Phonemetadata.java has two more fields to represent possible lengths of phone
+   numbers. Changed BuildMetadataFromXml (and the resultant jar) to alter the way
+   that metadata about possible-lengths information is consumed when constructing
+   metadata to populate these.
+   Discussion list email:
+   https://groups.google.com/forum/#!topic/libphonenumber-discuss/75TOpTFVi08
+Metadata changes:
```

Comment:
Merge them together. I just put them here because as Keghani said we don't have a place to put them, and I didn't think the script would automatically notice them since I actually updated all the files with my proto change. You can do it just here editing it in the UI if you want to, or you can manually edit in your client and then do git commit java/release_notes.txt and then git push.